stories/hits-per-page.stories.js

```diff
@@ -9,7 +9,7 @@ storiesOf('HitsPerPage', module)
         instantsearch.widgets.hitsPerPage({
           container,
           items: [
-            { value: 3, label: '3 per page' },
+            { value: 3, label: '3 per page', default: true },
```

Comment:
We can remove the example "with default hitPerPage to 5" it's redundant with the default one.

Reply:
Not really, it shows that the second option in the select can be the one selected by default.

Reply:
Yes... but I mean it's expected I think.